QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeThreeTotes.java

```diff
@@ -4,14 +4,14 @@
 import ccre.holders.TuningContext;
 import ccre.instinct.AutonomousModeOverException;
 
-public class AutonomousModeFull extends AutonomousModeBase {
+public class AutonomousModeThreeTotes extends AutonomousModeBase {
```

Comment:
Since the name of the class is changed, the names of published channels should be updated as well. See lines 55-58.